### PR TITLE
Update Auth Security; Modify Users Table

### PR DESCRIPTION
**Pull request by iamstevedavis**
Change to native crypto module instead of bcryptjs
Store password salt
Implement create user route (idempotent)
Email is now unique on user table

### CODE REVIEW

File: seeds/create_users.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
+const crypto = require("crypto");
```
Comment (clark-teeple) [inline seeds/create_users.js:1]: import statement for consistency

Comment (clark-teeple) [reply seeds/create_users.js:1]: oh, this is a seed file... no worries if it's a different ecma or whatever.

Comment (iamstevedavis) [reply seeds/create_users.js:1]: I don't think you can use imports here because these files aren't run after transpile

Comment (clark-teeple) [reply seeds/create_users.js:1]: yeah, makes sense.

**Pull request merged**